core/services.go

```diff
@@ -276,30 +284,38 @@ func (app *Application) createEvent(clientID string, current *model.User, eventI
 		return nil, err
 	}
 
+	var userIDs []string
 	var recipients []notifications.Recipient
 	if len(event.ToMembersList) > 0 {
-		recipients = event.GetMembersAsNotificationRecipients(skipUserID)
+		userIDs = event.GetMembersAsUserIDs(skipUserID)
 	} else {
-		result, _ := app.storage.FindGroupMemberships(clientID, model.MembershipFilter{
-			GroupIDs: []string{group.ID},
-			Statuses: []string{"member", "admin"},
-		})
-		recipients = result.GetMembersAsNotificationRecipients(skipUserID)
+		userIDs = []string{group.ID}
```

Comment:
Why are we putting `group.ID` into a variable called `userIDs`?

Reply:
Same as above